typings/externs.d.ts

```diff
@@ -63,6 +63,7 @@ declare global {
       onlyAudits?: string[] | null;
       onlyCategories?: string[] | null;
       skipAudits?: string[] | null;
+      skipNavigation?: boolean;
```

Comment:
our tests will hopefully force you to add this to `core/config/constants.js` as well